tests/sources/xml/test_marc.py

```diff
@@ -961,6 +961,269 @@ def test_get_call_numbers_transforms_correctly_if_fields_missing():
     assert Marc.get_call_numbers(source_record) is None
 
 
+def test_get_content_type_success():
+    source_record = create_marc_source_record_stub()
+    assert Marc.get_content_type(source_record) == ["Language material"]
+
+
+def test_get_content_type_transforms_correctly_if_char_position_blank():
+    source_record = create_marc_source_record_stub(
+        leader_field_insert="<leader>03282n m  2200721Ki 4500</leader>"
+    )
+    assert Marc.get_content_type(source_record) is None
```

Comment:
For tests involving values retrieved from fixed length fields, I wasn't sure how to (or if we can) depict "missing" tests. As shown in line 71, the whitespace between "n" and "m" (position 6 of the string) represents a blank / technically missing character.

Same thought for the [`get_dates` field method tests](https://github.com/MITLibraries/transmogrifier/pull/201/files#diff-1452e6d90ee026c11c6503638ab0a0ea9b7765e51e8409e451957967b2731defR1126-R1132).

Reply:
Agree that "missing" is basically impossible here!